src/api.js

```diff
@@ -4,11 +4,16 @@ const marketApi = axios.create({
     baseURL: 'https://kr-news-portfolio.onrender.com/api'
 })
 
-export const getArticles = ( )=> {
+export const getArticles = (slug )=> {
+    if (slug) { return marketApi.get(`/articles?topic=${slug}`).then((res)=> 
```

Comment:
We could look at using the one api call and handling the options in the api.js file. We don't actually need to make two separate api calls here.